pkg/scheduler/ugm/manager.go

```diff
@@ -477,13 +480,15 @@ func (m *Manager) clearEarlierSetUserLimits(newUserLimits map[string]map[string]
 // resetUserEarlierUsage Clear or reset earlier usage only when user already tracked for the queue path.
 // Reset the max apps and max resources to default, unlink the end leaf queue of queue path from its immediate parent and
 // eventually remove user tracker object itself from ugm if it can be removed.
-func (m *Manager) resetUserEarlierUsage(ut *UserTracker, hierarchy []string) {
+func (m *Manager) resetUserEarlierUsage(ut *UserTracker, queuePath string) {
 	// Is this user already tracked for the queue path?
+	hierarchy := strings.Split(queuePath, configs.DOT)
 	if ut.IsQueuePathTrackedCompletely(hierarchy) {
 		log.Log(log.SchedUGM).Debug("Need to clear earlier set configs for user",
 			zap.String("user", ut.userName),
 			zap.Strings("queue path", hierarchy))
 		ut.setLimits(hierarchy, nil, 0, false, false)
+		m.events.sendLimitSetForUser(ut.userName, queuePath)
```

Comment:
As discussed earlier, limits would be removed or reset to the default values. Earlier configured values are cleared now. So, limits removal event.

Reply:
All events are sent from the trackers from now on.